shared/validation/place-order-request-validator.ts

```diff
@@ -46,7 +46,7 @@ function validateItems(orderItems: OrderItemValidationObject[]): string[] {
 function validateOptions(orderItems: OrderItemValidationObject[]): string[] {
     let errors = [];
     for (let item of orderItems) {
-        if (item.options.length > 0) {
+        if (item.options.length > 0 && item.name !== "Potato Skins with your favourite topping") {
```

Comment:
This works, but when put together to the code below it begins pushing the boundaries of reasonable complexity and readability. At this point, I would look at having different validation rules and some kind of strategy pattern. So one strategy for when the item is a pizza, one for when it's specifically potato skins, and one generic rule that always gives an error if there are any options.